help_to_heat/frontdoor/routing/__init__.py

```diff
@@ -0,0 +1,34 @@
+from collections import deque
+
+from help_to_heat.frontdoor.consts import govuk_start_page, unknown_page
+from help_to_heat.frontdoor.routing.forwards_routing import get_next_page
+
+# in case of infinite loop ensure a route can't go on forever
+# currently the longest route is under 30 pages
+max_route_length = 100
+
+
+def get_route(answers, target_page, start_page=None):
```

Comment:
Think this could do with a comment explaining what it's producing and probably a rename: route being a loaded term (I'd expect a function with this name to return a relative path by default). Should `target` be `current` here, and `current_page` renamed to something else below? `target makes me think I'm trying to go somewhere.